src/pids.h

```diff
@@ -26,11 +44,24 @@ typedef struct
     int message_len;
     int message_displayed;
 
+    asd_t audio_services[MAX_AUDIO_SERVICES];
+    dsd_t data_services[MAX_DATA_SERVICES];
+
+    int parameters[NUM_PARAMETERS];
+
     char slogan[96];
     uint8_t slogan_have_frame[16];
     int slogan_encoding;
     int slogan_len;
     int slogan_displayed;
+
+    char alert[382];
```

Comment:
Make defines for these fixed values too just like MAX_DATA_SERVICES?  Nice for improving readability.

Reply:
Good idea. I'll do the same for the other magic numbers in pids.c and pids.h.